lib/index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function jwtDecode(token: string): any;
```

Comment:
this doesn't support the support the second paranmeter `options`

```
    declare module "jwt-decode" {
        interface OptionsType {
            header: boolean;
        }
        type jwtDecodeType = (token: string, options?: OptionsType) => any;
        const jwtDecode: jwtDecodeType;
        export default jwtDecode;
    }
```

Can be modified though